### ISSUE TITLE
GO ftp latest-full MySQL dump not available?

### ISSUE THREAD

**Issue by davidbio** [geneontology/helpdesk#4]
Dear GO-Team,

I was wondering what happened to the files in:
[http://archive.geneontology.org/full/latest/](http://archive.geneontology.org/full/latest/)
Currently March 2017.

The file assocdb-data only weights about 230 MB compared to 6.2 GB in January.
From the table sizes I get the impression that the "association" table is incomplete (March 2017).

The February folder is completely empty.

Am I missing something, or has the file's content changed?

I'd appreciate it if you could explain what happened here.

Best,
David

**Comment by kltm** [comment]
@davidbio,

Thank you for contacting the GO Helpdesk.

Our apologies for the trouble. There were apparently some issues with the final build for the GO full database. I would suggest using the 2017-01-01 build instead.
See #3.

It may be good to note that we will soon be switching over to a graph store and that the mysql dumps may not be produced in the future.

Best wishes, kltm

**Comment by kltm** [comment]
@stuartmiyasato

It may be good to pull the "bad" archive files to prevent further confusion.

**Comment by davidbio** [comment]
@kltm 
Thank you for the quick answer.

Where can I find this "Graph Store", what data-format is it and does it contain the same type of relationships?
e.g. dbxref -> gene_product -> association -> GO term?
Is there any documentation about this?

Best,
David

**Comment by kltm** [comment]
@davidbio, 

Well, we're in a bit of a transition period right now setting up the new pipeline, which will include the graphstore--we're no longer updating the mysql dumps in an effort to put that effort towards completing the graphstore. Our testing endpoint right now is at http://rdf.geneontology.org, with documentation (as it stands) at https://github.com/geneontology/go-graphstore/ (@cmungall may want to fill in more information here).

Best wishes. kltm

**Comment by stuartmiyasato** [comment]
>@stuartmiyasato
>
>It may be good to pull the "bad" archive files to prevent further confusion.

@kltm Just got back from vacation.  This is now done.

**Comment by kltm** [comment]
@stuartmiyasato Thank you! Welcome back!

**Comment by sgoetz** [comment]
As David mentioned above we saw that the GO MySql Dump "assoc_data" which contained all the consortium annotations is not getting updated anymore  (latest version is from January). 
These dumps had always been very useful for us and I have some questions regarding its continuity: 

- Are you planning to still create these dumps in the future?
- If not, could you provide support to populate the "old" scheme with the latest GAF files by ourselves?
- Will there be something similar to a dump for the new graph based RDF resource for the annotations/GAFs we could consider using in the future? 
- If yes, do you have any approximate timeline for this?

Thank you! 
Stefan

**Comment by kltm** [comment]
Apologies, this transition has been a long time in coming--the MySQL pipeline has taken an inordinate about of machine and time to maintain.

* The "full" dumps are no longer being produced; the various light dumps will be continued for a little while longer while the graph store workflow and data dumps are being worked out
* The current loads are produced by a quite old (and brittle) perl SQL system; if you are interested, we can perhaps arrange a tour of the repo and commands (all code necessary in a public VCS)
* Absolutely--before the plug is completely pulled on anything, we will make sure to provide a complete querying solution (we need this ourselves too)
* The transition is already under way, and will be completed in the next month or so

As a note, one of the (several) reasons that we're moving away from the GAF-based SQL system is that our current and future data models are much richer, meaning that the data we will be (and already are) capturing cannot be realized by GAF.

We are curently looking at offering Blazegraph dumps (journals) to fill the same gap. As well, we'll be offering a public RDF endpoint, as well as a GOOSE-like beginners query environment for common queries.

**Comment by davidbio** [comment]
This is just to let you know, that the latest folder points again to April and contains no valid data.
http://archive.geneontology.org/full/latest/

**Comment by kltm** [comment]
@stuartmiyasato Are parts of the "full" pipeline still ticking over enough to be causing the partial updates? That is something that we can definitely declare "done" and disassemble. A final manual step might be to aim latest at the January release and remove the bum March one.

**Comment by stuartmiyasato** [comment]
@kltm Huh, weird.  I had disabled the cron job that performs the load and I had commented out the job that copies the files to the archive site, but there was a duplicate copying line that was uncommented.  So the loading server apparently copied over some bogus build from a previous month.

I removed the uncommented line from the crontab and I manually fixed the "latest" link.  I also deleted the incomplete 2017-03-01 build.  I think that should take care of the issue.
